explainer.md

```diff
@@ -0,0 +1,802 @@
+
+The `.weekOfYear` property represents the ISO week-number. Beware that dates at the begining of a year may be part of a week from the preceding year, and dates at the end of a year may be part of a week at the beginning of the next year, as the first week of any year is defined as the week that contains the first Thursday of the week.
+
+#### OffsetDateTime.prototype.plus(data: IntervalLike): ZonedDateTime
+
+Creates a new `ZonedDateTime` object by adding (subtracting for negative values) values to its members. The specified values must be numeric if specified.
+
+The algorithm is such that:
+
+1. the individual values are added to the existing values.
+2. the range of `nanoseconds` is ensured to be between 0 and 999 by adjusting the `microseconds`
+3. the range of `microseconds` is ensured to be between 0 and 999 by adjusting the `milliseconds`
+4. the range of `milliseconds` is ensured to be between 0 and 999 by adjusting the `seconds`
+5. the range of `seconds` is ensured to be between 0 and 59 by adjusting the `minutes`
+6. the range of `minutes` is ensured to be between 0 and 59 by adjusting the `hours`
+7. the range of `hours` is ensured to be between 0 and 23 by adjusting `days`
+8. the range of `days` is ensured to be between 1 and 29-31 depending on the month by adjusting `month`
+9. the range of `months` is ensured to be between 1 and 12 by adjusting the `years`.
+
+#### OffsetDateTime.prototype.minus(other: ZonedDateTime): Interval
+#### OffsetDateTime.prototype.withZone(iana: string) : ZonedDateTime
+#### OffsetDateTime.prototype.toString() : string
+
+This creates an ISO-8601 string in the following format:
+```js
+${year}-${month}-${day}T${hours}:${minutes}:${seconds}.${nanoseconds}${offset}
+
+The `year` is 0-padded to a minimum of 4 digits. `month`, `day`, `hours`, `minutes`, `seconds` are 0-padded to a minimum of 2 digits. `nanoseconds` is 0-padded to a minimum of 9 digits. The `offset` is the timezone offset as created by `zoned.offset`.
+
+Examples:
+
+- `1976-11-18T15:23:30.123456789+01:00` - created with `+01:00` offset
+
+#### OffsetDateTime.prototype.toJSON() : string
+
+Equivalent to `OffsetDateTime.prototype.toString() : string`
+
+#### OffsetDateTime.fromString(iso: string) : ZonedDateTime
+
+Parses a `string` in the specific ISO-8601 format emitted by `OffsetDateTime.prototype.toString()`.
+
+#### OffsetDateTime.fromZonedDateTime(zoned : ZonedDateTime) : OffsetDateTime
+
+
+
+
+
+### ZonedDateTime
+
+An `ZonedDateTime` is an object that specifies a specific point in time with an *IANA timezone*.
+It bases this on an `Instant` and an iana `string`.
+
+#### ZonedDateTime(instant: Instant, zone: string) - constructor
+
+The constructor may only be called as such. It takes two arguments. The first is an instance
+of `Instant` and the second is a `string` representing a valid IANA timezone.
+
+#### ZonedDateTime.prototype.instant : BigInt
+
+The instance of `Instant` that was used to create this `ZonedDateTime`.
```

Comment:
Could we restructure this to eliminate the redundancy via text like "`ZonedDateTime.prototype` includes all properties and methods of `OffsetDateTime.prototype`, plus the following additions and replacements"?